feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.ts

```diff
@@ -147,8 +169,59 @@ export class AsmBindCartComponent implements OnInit, OnDestroy {
     this.subscription.add(subscription);
   }
 
+  onSaveInactiveCart() {
+    this.asmComponentService.setShowInactiveCartInfoAlert(false);
+    const customerId =
+      this.asmComponentService.getSearchParameter('customerId');
+    this.multiCartFacade.loadCart({
+      cartId: this.inactiveCartId,
+      userId: customerId,
+    });
+    this.multiCartFacade
+      .getCartEntity(this.inactiveCartId)
+      .pipe(
+        filter((state) => state.success === true),
+        take(1),
+        map((state) => state.value as Cart),
+        concatMap((cart) =>
+          iif(() => Boolean(cart !== null), this.openASMSaveCartDialog(cart))
+        )
+      )
+      .subscribe();
+
+    this.savedCartFacade
+      .getSaveCartProcessSuccess()
```

Comment:
you can subscribe error, success in one subscribe as below

this.service.getAll()
.subscribe(
    data => this.onSuccess(data),
    error => this.handleError(error),
    () => this.onComplete()
);

Reply:
fixed